tests/scoring/statistics/classification/tests/test_micro_macro_stats.py

```diff
@@ -6,17 +8,22 @@
 
 def test_micro_macro_stats():
     # (tp, fp, tn, fn)
-    stats = {
-        'Short': SPS(counts=(10, 2, 5, 8)),
-        'Labels': SPS(counts=(9, 3, 9, 4)),
-        'Can': SPS(counts=(11, 1, 8, 5)),
-        'Be': SPS(counts=(10, 2, 9, 4)),
-        'Columns': SPS(counts=(5, 7, 3, 10))
-    }
+    stats_keys = ['Short', 'Labels', 'Can', 'Be', 'Columns']
+    stats_values = [
+        SPS(counts=(10, 2, 5, 8)), 
+        SPS(counts=(9, 3, 9, 4)), 
+        SPS(counts=(11, 1, 8, 5)), 
+        SPS(counts=(10, 2, 9, 4)), 
+        SPS(counts=(5, 7, 3, 10))
+    ]
+    stats = OrderedDict()
+    for key,value in zip(stats_keys, stats_values):
+        stats[key] = value
```

Comment:
No big deal.  In the future, I might follow this pattern:
```
    stats_keys = ['Short', 'Labels', 'Can', 'Be', 'Columns']
    stats_values = [
        SPS(counts=(10, 2, 5, 8)), 
        SPS(counts=(9, 3, 9, 4)), 
        SPS(counts=(11, 1, 8, 5)), 
        SPS(counts=(10, 2, 9, 4)), 
        SPS(counts=(5, 7, 3, 10))
    ]
    stats = OrderedDict(zip(stats_keys, stats_values))
```

Or even keep it easier to follow the keys and values with this pattern:
```
    stats = OrderedDict([
      ('Short', SPS(counts=(10, 2, 5, 8))),
      ('Labels', SPS(counts=(9, 3, 9, 4))),
      ...])
```